components/calendar/overview.md

```diff
@@ -71,6 +71,37 @@ You can display a wider range of dates by rendering multiple instances of the Ca
 
 The Calendar generates events that you can handle and further customize ist behavior. [Read more about the Blazor Calendar events...]({%slug components/calendar/events%}).
 
+## Calendar Parameters
+
+The Blazor Calendar provides various parameters that allow you to configure the component. Also check the [Calendar's public API](/blazor-ui/api/Telerik.Blazor.Components.TelerikCalendar).
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Attribute | Type and Default Value | Description |
+|----------|----------|----------|
+| `BottomView` | ` CalendarView` <br/> `CalendarView.Month` | The bottommost view in the Calendar to which the user can navigate to. |
+| `Date` | `DateTime` | The date to which the user has navigated the Calendar. Supports two-way binding. |
```

Comment:
"the user has navigated" - not necessarily. The developer has control over this too. We could explain the parameter in another way - for example, this is the date that will certainly belong to the current view?